zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -275,6 +275,51 @@ def _do_cleanup(self, metadataPath):
+                continue
+
+            metadata_path = os.path.join(metadataDir, fname)
+            with self._get_path_lock(metadata_path):
+                removed_any = False
+                for path in [metadata_path, metadata_path + '.tmp',
+                             metadata_path + '.summary', metadata_path + '.summary.tmp']:
+                    try:
+                        if os.path.exists(path):
+                            os.remove(path)
+                            removed_any = True
+                    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)